diff --git a/autoload/SpaceVim/autocmds.vim b/autoload/SpaceVim/autocmds.vim
index bfea9872..57cb1af7 100644
--- a/autoload/SpaceVim/autocmds.vim
+++ b/autoload/SpaceVim/autocmds.vim
@@ -52,7 +52,7 @@ function! SpaceVim#autocmds#init() abort
     autocmd Filetype qf setlocal nobuflisted
     autocmd FileType python,coffee call SpaceVim#util#check_if_expand_tab()
     au StdinReadPost * call s:disable_welcome()
-    autocmd InsertEnter * call s:fixindentline()
+    " autocmd InsertEnter * call s:fixindentline()
     autocmd BufEnter,FileType * call SpaceVim#mapping#space#refrashLSPC()
     if executable('synclient') && g:spacevim_auto_disable_touchpad
       let s:touchpadoff = 0
diff --git a/autoload/SpaceVim/lsp.vim b/autoload/SpaceVim/lsp.vim
index 2ffdba7e..97fd9c25 100644
--- a/autoload/SpaceVim/lsp.vim
+++ b/autoload/SpaceVim/lsp.vim
@@ -63,7 +63,8 @@ if SpaceVim#layers#isLoaded('autocomplete') && get(g:, 'spacevim_autocomplete_me
   endfunction
 
   function! SpaceVim#lsp#go_to_def() abort
-    call CocAction('jumpDefinition')
+	execute "norm! g\<c-]>"
+    " call CocAction('jumpDefinition')
   endfunction
 
   function! SpaceVim#lsp#go_to_declaration() abort
